src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String CSV_SPLIT_BY = ",";
 
+    public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
It is necessary to break the logic from this method into 3 separate methods:
1. Read data from file
2. We process this data, i.e. create a report
3. Write the report to a file